packages/cozy-harvest/src/Manifest.spec.js

```diff
@@ -0,0 +1,136 @@
+/* eslint-env jest */
+import { Manifest } from './Manifest'
+
+describe('Manifest', () => {
+  describe('sanitize', () => {
+    it('should remove "fields" if fields is null', () => {
+      const manifest = {
+        fields: null
```

Comment:
What happen when manifest is `undefined`? I guess we should have the case of an undeclared `fields` property more often than a `fields` property having a `null` value, and the test should be better with `const manifest = {}`. What do you think ?

My bad : the test is right after 👍